src/main/java/edu/utdallas/davisbase/representation/DeleteCommandRepresentation.java

```diff
@@ -0,0 +1,33 @@
+package edu.utdallas.davisbase.representation;
+
+public class DeleteCommandRepresentation implements CommandRepresentation {
```

Comment:
This class is missing the following getter methods:

- `public String getTable()`
- `public WhereExpression getWhereExpression()`